exporter/splunkhecexporter/client.go

```diff
@@ -204,34 +202,44 @@ func (c *client) fillLogsBuffer(logs plog.Logs, buf buffer, is iterState) (iterS
 				is.record = 0 // Reset record index for next library.
 				logRecord := sl.LogRecords().At(k)
 
+				var err error
+				var l int
 				if c.config.ExportRaw {
-					b = []byte(logRecord.Body().AsString() + "\n")
+					_, err = buf.Write([]byte(logRecord.Body().AsString() + "\n"))
```

Comment:
nit: if there is an `errOverCapacity` here the error message generated on line 241 will say "... event size 0 bytes larger than ..."

Reply:
Good point thanks!